code/modules/reagents/chemistry/reagents/pyrotechnic.dm

```diff
@@ -43,6 +45,7 @@
 	reagent_state = SOLID
 	color = "#673910" // rgb: 103, 57, 16
 	process_flags = ORGANIC | SYNTHETIC
+	taste_message = "metal"
```

Comment:
I can assure you rust is quite a potent flavor. There's a reason you can taste it in water without visible discoloration. also of note, we need iron to live, that's why we're so good at tasting it in things. Aluminum, pretty flavorless, tastes like whatever is on it really. That's all that's really in thermite, so rusty metal would be relatively accurate all things considered. And no, I don't go around licking metal in my spare time, but when you work with it for a long time, it eventually finds its way into your mouth.